tasks/internal/get-advisory-severity/tests/test-get-advisory-severity-component.yaml

```diff
@@ -13,7 +13,7 @@ spec:
         name: get-advisory-severity
       params:
         - name: releaseNotesImages
-          value: '[{"repository":"component","cves":{"fixed":{"CVE-moderate":{"components":[]}}}}]'
```

Comment:
Can you add comments like you did here https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/internal/create-advisory-task/tests/test-create-advisory.yaml#L17?

Reply:
done